src/basic/constants.h

```diff
@@ -87,3 +87,5 @@
 
 /* Recommended baseline - see README for details */
 #define KERNEL_BASELINE_VERSION "5.7"
+
+#define PID_AUTOMATIC ((pid_t) INT_MIN) /* special value indicating "acquire pid from connection peer" */
```

Comment:
sorry, but I'm still not convinced. Feels definitely confusing to me to stuff a special identifier value into "well-known constants"

Reply:
Inlined the value of the constant for now instead.